themes/classic/modules/ps_linklist/views/templates/hook/linkblock-column.tpl

```diff
@@ -0,0 +1,56 @@
+{**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ *}
+<div class="links">
+  {foreach $linkBlocks as $linkBlock}
+    <div class="wrapper">
+      <p class="h3 hidden-sm-down">{$linkBlock.title}</p>
+      {assign var=_expand_id value=10|mt_rand:100000}
+      <div class="title clearfix hidden-md-up" data-target="#footer_sub_menu_{$_expand_id}" data-toggle="collapse">
```

Comment:
You can use   `$linkBlock.id`  instead of random value?

Reply:
Copy pasted the basic template, with removing bootstrap classes, I think if we wants to improve the module, let's doing it on another PR with another issue :)

Reply:
As I understand it, here this is an override, so I think we can fix that here.